File: ProblemSet1/Mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;

    do
    {
        n = get_int("size: ");
    }
    while (n < 1)

    for (int i = 0; i < n; i++)
    {
        for (int j = n - 1; j >= 0; j--)
        {
            if (j > i)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("  ");
        for (int k = 0; k < n; k++)
        {
            if (k <= i)
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: ProblemSet6/sentimental-mario-more/mario.py
from cs50 import get_int

# Prompt user for height. Check whether input is positive integer and reprompt user if not
while True:
    n = get_int("Height: ")
    if 1 <= n <= 8:
        break
    else:
        print("Please enter a number between 1 and 8")

# Three loops, one outer to handle the rows and two inner to handle each place within the rows
for i in range(n):
    for j in range(n - 1, -1, -1):
        if j > i:
            print(" ", end="")
        else:
            print("#", end="")
    print("  ", end="")
    for k in range(n):
        if k > i:
            print("", end="")
        else:
            print("#", end="")
    print()
